admin/database/postgres/postgres.go

```diff
@@ -657,11 +655,11 @@ func (c *connection) CountSingleuserOrganizationsForMemberUser(ctx context.Conte
 	return count, nil
 }
 
-func (c *connection) FindProjectMemberUsers(ctx context.Context, projectID string) ([]*database.Member, error) {
+func (c *connection) FindProjectMemberUsers(ctx context.Context, projectID string, paginationOpts *database.PaginationOptions) ([]*database.Member, error) {
 	var res []*database.Member
 	err := c.getDB(ctx).SelectContext(ctx, &res, `SELECT u.id, u.email, u.display_name, u.created_on, u.updated_on, r.name FROM users u 
     	JOIN users_projects_roles upr ON u.id = upr.user_id
-		JOIN project_roles r ON r.id = upr.project_role_id WHERE upr.project_id=$1`, projectID)
+		JOIN project_roles r ON r.id = upr.project_role_id WHERE upr.project_id=$1 AND lower(u.email) > $2 ORDER BY lower(u.email) limit $3`, projectID, paginationOpts.Cursor, paginationOpts.PageSize)
```

Comment:
1. Capitalize SQL keywords
2. Also please fix odd line breaks/indentation in this query